lib/smart_app_launch/smart_access_brands_suite.rb

```diff
@@ -0,0 +1,59 @@
+require_relative 'smart_access_brands_group'
+
+module SMARTAppLaunch
+  class SMARTAccessBrandsSuite < Inferno::TestSuite
+    id 'smart_access_brands'
+    title 'SMART Access Brands STU2.2'
+    version VERSION
+
+    description <<~DESCRIPTION
+      The SMART User-access Brands Test Suite verifies that Brand Bundle Publishers publish valid User-access
+      Brand Bundles according to the SMART App Launch IG
+      [User-access Brands and Endpoints](https://build.fhir.org/ig/HL7/smart-app-launch/brands.html#user-access-brands-and-endpoints)
+      requirements.
+
+      The specification defines FHIR profiles for Endpoint, Organization, and Bundle resources that help users connect
+      their apps to health data providers. It outlines the process for data providers to publish FHIR Endpoint and
+      Organization resources, where each Organization includes essential branding information such as the organization's
+      name, logo, and user access details. Apps present branded Organizations to help users select the right data
+      providers.
+
+      This Test Suite provides validation testing to ensure the published User-access Brands Bundles are valid and
+      contain valid resources.
+    DESCRIPTION
+
+    VALIDATION_MESSAGE_FILTERS = [
+      /\A\S+: \S+: URL value '.*' does not resolve/,
+      %r{\A\S+: \S+: Bundled or contained reference not found within the bundle/resource} # Validator issue with Brand profile: https://chat.fhir.org/#narrow/stream/291844-FHIR-Validator/topic/SMART.20v2.2E2.20User.20Access.20Brands.3A.20Brand.20validation.20error.3F/near/466321024
+    ].freeze
+
+    fhir_resource_validator do
+      igs 'hl7.fhir.uv.smart-app-launch#2.2.0'
+
```

Comment:
Add this in here (per [inferno docs](https://inferno-framework.github.io/inferno-core/docs/Inferno/DSL/FHIRResourceValidation/Validator.html) and [wrapper docs](https://github.com/hapifhir/org.hl7.fhir.validator-wrapper/blob/0093f0b02b9b88c86cf4e1da75d4aed410acf8d5/src/commonMain/resources/static-content/openapi.yml#L218C13-L218C29)):

```ruby
cli_context({
  # Allow example URLs because we give tester option to follow URLs anyhow
  # (configurable) and its nice to be able to have the examples in the IG pass
  allowExampleUrls: true
})
```

Reply:
That will get rid of this on example2 from the IG
<img width="753" alt="Screenshot 2024-10-01 at 4 17 06 PM" src="https://github.com/user-attachments/assets/e944f9c5-e132-4efe-a471-e7ec4e8a458c">